tests/PySys/software-management-end-to-end/sm-apt-install-fail/run.py

```diff
@@ -0,0 +1,36 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate end to end behaviour for failing installation
+
+When we install a package that cannot be installed with the apt package manager
+Then we receive a failure from the apt plugin
+"""
+
+import json
+import requests
+import time
+import sys
+
+sys.path.append("software-management-end-to-end")
+from environment_sm_management import SmManagement
+
+
+class PySysTest(SmManagement):
+    def setup(self):
+        super().setup()
+        self.assertThat("False == value", value=self.check_isinstalled("rolldice"))
```

Comment:
What is the purpose of this check?

Reply:
That line did not make sense here, so I removed it